### PR TITLE
Update AWSBucketAPILogs-SuspiciousDataAccessToS3BucketsfromUnknownIP.…

### PR DESCRIPTION
**Pull request by referefref**
…yaml

Adjusted spacing on parameters on line 25 for consistency.

   Required items, please complete
   
   Change(s):
   - Adjusted parameter spacing on AWSBucketAPILogs-SuspiciousDataAccessToS3BucketsfromUnknownIP.yaml line 25

   Reason for Change(s):
   - Consistency and readability of query

   Version Updated:
   - 1.0.0

   Testing Completed:
   - Yes

   Checked that the validations are passing and have addressed any issues that are present:
   - Yes

# Guidance <- remove section before submitting
-----------------------------------------------------------------------------------------------------------
## Before submitting this PR please ensure that you have read the following sections and filled out the changes, reason for change and testing complete sections:

Thank you for your contribution to the Microsoft Sentinel Github repo.

> Details of the code changes in your submitted PR.  Providing descriptions for pull requests ensures there is context to changes being made and greatly enhances the code review process.  Providing associated Issues that this resolves also easily connects the reason.
   
   Change(s):
   - Updated syntax for XYZ.yaml

   Reason for Change(s):
   - New schema used for XYZ.yaml
   - Resolves ISSUE #1234

   Version updated:
   - Yes
   - Detections/Analytic Rule templates are required to have the version updated

> The code should have been tested in a Microsoft Sentinel environment that does not have any custom parsers, functions or tables, so that you validate no incorrect syntax and execution functions properly.  If your submission requires a custom parser or function, it must be submitted with the PR.

   Testing Completed:
   - Yes/No/Need Help

_Note: If updating a detection, you must update the version field._

> Before the submission has been made, please look at running the KQL and Yaml Validation Checks locally.
> https://github.com/Azure/Azure-Sentinel#run-kql-validation-locally

   Checked that the validations are passing and have addressed any issues that are present:
   - Yes/No/Need Help
   
   _Note: Let us know if you have tried fixing the validation error and need help._

> References: 
> - [Guidance for Detection checks](https://github.com/Azure/Azure-Sentinel#pull-request-detection-template-structure-validation-check)
> - [General contribution guidance](https://github.com/Azure/Azure-Sentinel/wiki#what-can-you-contribute-and-how-can-you-create-contributions)
> - [PR validation troubleshooting](https://github.com/Azure/Azure-Sentinel#pull-request)

-----------------------------------------------------------------------------------------------------------

### CODE REVIEW

**Comment by v-mchatla** [comment]
Hi @referefref,
I will be reviewing this today and let you know if I need any help.
Thanks

File: Hunting Queries/AWSS3/AWSBucketAPILogs-SuspiciousDataAccessToS3BucketsfromUnknownIP.yaml [hunk 1]
```diff
@@ -22,7 +22,7 @@ query: |
   AwsBucketAPILogs_CL 
   | where EventTime TimeGenerated between(starttime..endtime)
```
Comment (v-mchatla) [inline Hunting Queries/AWSS3/AWSBucketAPILogs-SuspiciousDataAccessToS3BucketsfromUnknownIP.yaml:23]: Hi @referefref, 
On what column you are applying filter EventTime or TimeGenerated. Can you please check and confirm.
Thanks

**Comment by v-mchatla** [comment]
Hi @referefref,
Can you please address the query.
Thanks

File: Hunting Queries/AWSS3/AWSBucketAPILogs-SuspiciousDataAccessToS3BucketsfromUnknownIP.yaml (hunk 1 above)
Comment (referefref) [reply Hunting Queries/AWSS3/AWSBucketAPILogs-SuspiciousDataAccessToS3BucketsfromUnknownIP.yaml:23]: Hi,
This line of the query was not modified at all from the original.
Thanks.

**Comment by v-mchatla** [comment]
Hi @referefref,
Can you please address the query.
Thanks

**Comment by referefref** [comment]
Hi,

As above that line was not modified from the original, the spacing of the query parameters was adjusted for consistency, the time parameters and this line have not been modified at all.

Thanks,
James.

**Comment by v-mchatla** [comment]
Hi @referefref,
I agree that you haven't made any changes to that line but while testing it's returning error as we are applying between check on two different columns which is not allowed. As you are making some changes to it can you please fix that as well.
Thanks

**Comment by v-mchatla** [comment]
Hi @referefref,
I'm requesting you to remove the TimeGenerated from line 23 so that the query will get execute without any issues. I will approve and merge the PR once that is done. 
Thanks

**Comment by referefref** [comment]
> Hi @referefref, I'm requesting you to remove the TimeGenerated from line 23 so that the query will get execute without any issues. I will approve and merge the PR once that is done. Thanks

I have made that change as requested.

**Comment by v-atulyadav** [comment]
Hi @referefref, Thank you for this fix. We will check and update you soon.

**Review by v-mchatla — DISMISSED** [review]
Minor changes to the query.

**Comment by v-mchatla** [comment]
Hi @devikamehra,
Can you please take a look at it once and share your feedback if any.
Thanks

**Comment by v-atulyadav** [comment]
Hi @devikamehra,
Can you please take a look at it once and share your feedback if any.
Thanks

**Comment by v-mchatla** [comment]
Hi @referefref,
Sorry for the delay in response. I will check with Devika and get back to you.
Thanks

File: Hunting Queries/AWSS3/AWSBucketAPILogs-SuspiciousDataAccessToS3BucketsfromUnknownIP.yaml [hunk 2]
```diff
@@ -20,9 +20,9 @@ query: |
   let endtime = todatetime('{{EndTimeISO}}');
   let lookback = starttime - 14d;
   AwsBucketAPILogs_CL 
-  | where EventTime TimeGenerated between(starttime..endtime)
+  | where EventTime between(starttime..endtime)
```
Comment (devikamehra) [inline Hunting Queries/AWSS3/AWSBucketAPILogs-SuspiciousDataAccessToS3BucketsfromUnknownIP.yaml:23]: Is this query working? According to the schema of AwsBucketAPILogs_CL, EventName field will not be there. I think we meant AWSS3BucketAPILogParsed in first part of join as well.

**Review by devikamehra — CHANGES_REQUESTED** [review]

**Comment by v-mchatla** [comment]
Hi @referefref,
Will you be able to fix that as well. If not, I will take help of Devika for fixing it.
Thanks

**Comment by referefref** [comment]
> Hi @referefref, Will you be able to fix that as well. If not, I will take help of Devika for fixing it. Thanks

I have applied the fix to the latest patch.

**Comment by v-vdixit** [comment]
Hi @referefref we are looking into this will update you soon.

**Comment by v-mchatla** [comment]
Hi @devikamehra,
Can you please review the changes and provide your feedback.
Thanks

**Comment by v-mchatla** [comment]
Hi @devikamehra,
Author has made requested changes. Can you please review and share your feedback
Thanks

**Comment by v-mchatla** [comment]
Hi @devikamehra,
Author has made requested changes. Can you please review and share your feedback
Thanks

**Review by devikamehra — APPROVED** [review]

**Review by v-mchatla — APPROVED** [review]

**Review by v-dvedak — APPROVED** [review]

**Pull request merged**